programs/manifest/src/program/processor/create_market.rs

```diff
@@ -104,10 +104,14 @@ pub(crate) fn process_create_market(
             }
         }
 
-        // Do not need to initialize with the system program because it is assumed
-        // that it is done already and loaded with rent.
+        // Do not need to initialize with the system program because it is
```

Comment:
We need to find a solution for reducing the chances this happens accidentally though. Outside of the core is fine

Reply:
Just dont include it in the sdk, will have to know what you are doing to make one